alacritty_terminal/src/tty/windows/switchable_backends.rs

```diff
@@ -0,0 +1,188 @@
+/// Types to wrap either Conpty or Winpty backends.
```

Comment:
The name `switchable_backends` sounds strange to me. Why did you choose this name?

Reply:
Agreed, I don't love the name either. I considered alternatives like `dynamic_backend` but it doesn't use dynamic dispatch so didn't really like that much either.

Another option is to just put all of this stuff in the `winpty.rs` file, as it's only needed when the `winpty` feature is set.

Reply:
Why did you change stuff around so much? To prevent dead code warnings?

Reply:
Yes. An alternative would be to undo this move (put it back in `mod.rs`), and just add `#[cfg(feature = "winpty")]` to a lot more of the code there instead. Would that be preferable?